parser/ParseStatement.hpp

```diff
@@ -562,6 +566,43 @@ class ParseStatementInsert : public ParseStatement {
     return relation_name_.get();
   }
 
+ private:
+  std::unique_ptr<ParseString> relation_name_;
```

Comment:
I think we can skip this comment and keep `relation_name_` private. @zuyu mentioned that he agrees with this view. Thanks!